bfe_tls/tls.go

```diff
@@ -81,6 +81,7 @@ func NewListener(inner net.Listener, config *Config) net.Listener {
 	return l
 }
 
+// UpdateListener
 /* UpdateListener - update config for tls listener
```

Comment:
this line should be deleted

Reply:
according to [comment-sentences](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences), it should start with two backslashes.